napari/utils/notebook_display.py

```diff
@@ -86,7 +87,13 @@ def _clean_alt_text(self, alt_text):
                 str(alt_text)
             )  # cleaner won't recognize unescaped script tags
             cleaner = Cleaner()
-            doc = document_fromstring(alt_text)
+            try:
+                doc = document_fromstring(alt_text)
+            except ParserError:
+                alt_text = ""
+                warn(
+                    'The provided alt text does not constitute valid html, so it was discarded.'
+                )
```

Comment:
```suggestion
		stacklevel=3,
                )
```
To raise a warning in the place where the constructor of `NotebookScreenshot` is called. 

Or if public API just uses this class please increase this number. 

It may be worth adding `sctacklevel` to the previous warning about the lack of lxml

Reply:
I see, I never quite understood the stacklevel argument (I always reused warnings from the same code area :P).